plugins/modify-secret.yaml

```
@@ -0,0 +1,40 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: modify-secret
+spec:
+  version: v0.0.3
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/rajatjindal/kubectl-modify-secret/releases/download/v0.0.3/darwin-amd64-v0.0.3.tar.gz
+    sha256: 809327469859fa665a6944e73975e0d0542c997ab048b091afc725b8e6413438
+    files:
+    - from: "*"
+      to: "."
+    bin: kubectl-modify-secret
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/rajatjindal/kubectl-modify-secret/releases/download/v0.0.3/linux-amd64-v0.0.3.tar.gz
+    sha256: 9450d2c544859c18ad881e805d9fd9c3ec5bad5f82f6d6fba187f2e29706f672
+    files:
+    - from: "*"
+      to: "."
+    bin: kubectl-modify-secret
+  shortDescription: kubectl-modify-secrets allows user to directly modify the secret without worrying about base64 encoding/decoding
+  caveats: |
+    Usage:
+      kubectl modify_secret secret-name -n kube-system --kubeconfig /path/to/kube/config
```

```suggestion
      kubectl modify-secret secret-name -n kube-system --kubeconfig /path/to/kube/config
```
Krew only allows single-word plugin names. So your plugin will become `modify-secret` (and not `modify_secret`).

---

it seems like it uses underscores now instead of dashes for some reason. 

https://github.com/kubernetes-sigs/krew/blob/35dcafba7cf3d8f94f1c310a6f6c3bb685a8f8ab/docs/DEVELOPER_GUIDE.md#specifying-plugin-executable

personally I would have preferred it to behave like kubectl plugin naming behavior, but i am sure there must be a good reason for not doing that. (the usage would have been kubectl modify secret .......)

I will make other suggested changes soon, and will update the PR. (all of them are great suggestions, thanks for taking out time for that).

---

I am a little confused by this. Updated the manifest to use 'modify-secret'